app/Http/Controllers/PostsController.php

```diff
@@ -71,6 +71,11 @@ public function index(Request $request)
         $query = $this->newQuery(Post::class)
             ->orderBy('created_at', 'desc');
 
+        if ($request->query('include') && has_include($request, 'signup')) {
```

Comment:
Let's embed this `$request->query('include')` check in the helper as well, so this is just `if (has_include(...))`!

Reply:
oh yes that is nice, thanks @DFurnes !